### PR TITLE
Frontend: Added okShadow field to Profile / Reg pages

### PR DESCRIPTION
**Pull request by apluscs**
Screenshot 2020-07-31 at 9 21 41 AM
Screenshot 2020-07-31 at 9 22 14 AM

### CODE REVIEW

File: src/main/webapp/register.html [hunk 1]
```diff
@@ -160,6 +160,13 @@ <h5 class="text-center lead">Fill in the following fields below.</h5>
                 value="" name="linkedin">
           </div>
         </div>
+        <div class="form-check">
+          <input class="form-check-input" type="checkbox" id="ok-shadow-check">
+          <label class="form-check-label" for="ok-shadow-check">
+            I am OK if another person shadows the interviews I have.
```
Comment (meagancheese) [inline src/main/webapp/register.html]: Weird that this is worded differently than on the profile page.

**Review by meagancheese — CHANGES_REQUESTED** [review]
Make wording consistent and add pics to description.

File: src/main/webapp/register.html (hunk 1 above)
Comment (apluscs) [reply src/main/webapp/register.html]: Good catch

**Review by meagancheese — APPROVED** [review]

File: src/main/webapp/profile.html [hunk 2]
```diff
@@ -161,6 +161,14 @@ <h1 class="text-center">Your Profile</h1>
                 placeholder="" value="" name="linkedin">
           </div>
         </div>
+        <div class="form-check">
+          <input disabled class="form-check-input check-editable" type="checkbox" 
+              id="ok-shadow-check">
+          <label class="form-check-label" for="ok-shadow-check">
+            I am OK if another person shadows my interviews.
```
Comment (GrantComm) [inline src/main/webapp/profile.html]: Could this be "Allow another person to shadow my interviews" instead?

**Review by GrantComm — APPROVED** [review]
Just one question.

**Review by acmcarther — APPROVED** [review]

File: src/main/webapp/profile.html (hunk 2 above)
Comment (apluscs) [reply src/main/webapp/profile.html:168]: Good idea. Will change.

Comment (apluscs) [reply src/main/webapp/profile.html]: Good idea, will change.

**Pull request merged**